### PR TITLE
Homework: NWD, NWW

### PR DESCRIPTION
**Pull request by runer53248**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: nwd-nnw
🏅 Granted 6 XP for @runer53248

**Comment by ziobron** [comment]
⏱ 3 XP extra for @runer53248 for on-time delivery

File: homework/nwd-nnw/nwdNww.hpp [hunk 1]
```diff
@@ -1,11 +1,42 @@
...
+    if(rhs < 0){
+        rhs = -rhs;
+    }
+    if(lhs == rhs){
+        return lhs;
+    }
+    int max = rhs;
+    if(lhs > rhs){
+        max = lhs;
+    }
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:16]: Use std::abs 
and
Use std::swap

if (lhs > rhs) {
   std::swap(lhs, rhs)
}

File: homework/nwd-nnw/nwdNww.hpp [hunk 2]
```diff
@@ -1,11 +1,42 @@
...
+    if(lhs == 0 or rhs == 0){
+        return lhs + rhs;
+    }
+    int result = 1;
+    for(int i = max/2; i>1; --i){
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:21]: Add space between operations like '+' '-' '!=' '==' etc...

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**